ming/datastore.py

```diff
@@ -191,3 +203,53 @@ def db(self) -> Database:
 
             self._db = self.bind[self.name]
         return self._db
+
+    @property
+    def encryption(self) -> encryption.EncryptionConfig | None:
+        return self._encryption_config
+
+    @LazyProperty
+    def encryptor(self) -> ClientEncryption:
+        """Creates and returns a :class:`pymongo.encryption.ClientEncryption` instance for the given ming datastore. It uses this to handle encryption/decryption using pymongo's native routines.
+
+        :param ming_ds: the :class:`ming.datastore.Datastore` for which this encryptor should be configured with.
```

Comment:
can delete this line, since `self` is the datastore now!

Reply:
Fixed